tools/dgeni/templates/componentGroup.template.html

```diff
@@ -21,7 +21,7 @@
 
 <div class="docs-api">
   <h2>
-    API reference for Angular Material {$ doc.name $}
+    API reference for Angular {$ doc.packageDisplayName $} {$ doc.displayName $}
   </h2>
 
   <p class="docs-api-module-import">
```

Comment:
We should fix the import statement here as well. We can just use `doc.packageName` for the imports package name.

Reply:
Done